python/django/security/templates/injection-template.yaml.j2

```diff
@@ -1,217 +1,27 @@
-          - pattern: |
-              $DATA = request.$W.get($X)
-              ...
-              $RES = {{ sink }}($A.$C(..., $DATA, ...), ...)
-          - pattern: |
-              $DATA = request.$W[$X]
-              ...
-              $RES = {{ sink }}($A.$C(..., $DATA, ...), ...)
-          - pattern: |
-              $DATA = request.$W($X)
-              ...
-              $RES = {{ sink }}($A.$C(..., $DATA, ...), ...)
-          - pattern: |
-              $DATA = request.$W
-              ...
-              $RES = {{ sink }}($A.$C(..., $DATA, ...), ...)
-          - pattern: |
-              $DATA = request.$W.get($X)
-              ...
-              return {{ sink }}($A.$C(..., $DATA, ...), ...)
-          - pattern: |
-              $DATA = request.$W[$X]
-              ...
-              return {{ sink }}($A.$C(..., $DATA, ...), ...)
-          - pattern: |
-              $DATA = request.$W($X)
-              ...
-              return {{ sink }}($A.$C(..., $DATA, ...), ...)
-          - pattern: |
-              $DATA = request.$W
-              ...
-              return {{ sink }}($A.$C(..., $DATA, ...), ...)
-          - pattern: |
-              $DATA = request.$W.get($X)
-              ...
-              {{ sink }}(f"...{$DATA}...", ...)
-          - pattern: |
-              $DATA = request.$W[$X]
-              ...
-              {{ sink }}(f"...{$DATA}...", ...)
-          - pattern: |
-              $DATA = request.$W($X)
-              ...
-              {{ sink }}(f"...{$DATA}...", ...)
-          - pattern: |
-              $DATA = request.$W
-              ...
-              {{ sink }}(f"...{$DATA}...", ...)
-          - pattern: |
-              $DATA = request.$W.get($X)
-              ...
-              $RES = {{ sink }}(f"...{$DATA}...", ...)
-          - pattern: |
-              $DATA = request.$W[$X]
-              ...
-              $RES = {{ sink }}(f"...{$DATA}...", ...)
-          - pattern: |
-              $DATA = request.$W($X)
-              ...
-              $RES = {{ sink }}(f"...{$DATA}...", ...)
-          - pattern: |
-              $DATA = request.$W
-              ...
-              $RES = {{ sink }}(f"...{$DATA}...", ...)
-          - pattern: |
-              $DATA = request.$W.get($X)
-              ...
-              return {{ sink }}(f"...{$DATA}...", ...)
-          - pattern: |
-              $DATA = request.$W[$X]
-              ...
-              return {{ sink }}(f"...{$DATA}...", ...)
-          - pattern: |
-              $DATA = request.$W($X)
-              ...
-              return {{ sink }}(f"...{$DATA}...", ...)
-          - pattern: |
-              $DATA = request.$W
-              ...
-              return {{ sink }}(f"...{$DATA}...", ...)
-    message: |
-      {{ message | indent(width=8)  }}
-    languages: [python]
-    severity: ERROR
+            {{ left_pattern }}{{ sink }}(..., $INTERM, ...)
+    {%- endfor -%}
+  {%- endfor -%}
+{%- endfor -%}
```

Comment:
can you make sure there is newline at the end?

Reply:
Roger o7